src/garage/tf/core/parameter.py

```diff
@@ -26,6 +27,7 @@ def parameter(input_var,
         dtype: Data type of the variables (default is tf.float32).
         trainable (bool): Whether these variables are trainable.
         name (str): Variable scope of the variables.
+        pixel_env (bool): Whether the environment is pixel based or not.
```

Comment:
i think you forgot to update the documentation here.

Reply:
oops, will change that.